main.tf

```diff
@@ -56,7 +56,7 @@ locals {
   }
 }
 
-resource "aws_dynamodb_table" "basic-dynamodb-table" {
+resource "aws_dynamodb_table" "table" {
```

Comment:
Pretty sure this change would trigger a resource replacement. Is this acceptable?

Reply:
it is.  Best practices state dashes should be avoided in logical resource names, so simplifying the name removed the offending characters